### PR TITLE
Add theme helper, theme toggle component, docs

### PR DESCRIPTION
**Pull request by louisescher**
#### Description

- Adds a `ThemeHelper` class that can be used to easily manage the color scheme.
- Adds a pre-made `<ThemeToggle />` wrapper component with slots to create a theme toggle with.
- Adds docs for everything mentioned above.

### CODE REVIEW

**Comment by studiocms-no-reply** [comment]
Thank you for submitting your Pull Request, the following links will become available for preview shortly:


  - [StudioCMS Landing Page](https://pr377.www.studiocms.xyz/)

  - [StudioCMS Documentation](https://pr377.docs.studiocms.xyz/)

  - [StudioCMS Demo (node-playground)](https://pr377.demo.studiocms.xyz/) (Also deployed with package changes)

  - [StudioCMS Ui Demo (ui-playground)](https://pr377.ui-testing.studiocms.xyz/)

**Comment by studiocms-no-reply** [comment]
Thank you for submitting your Pull Request, the following links will become available for preview shortly:


  - [StudioCMS Landing Page](https://pr377.www.studiocms.xyz/)

  - [StudioCMS Documentation](https://pr377.docs.studiocms.xyz/)

  - [StudioCMS Demo (node-playground)](https://pr377.demo.studiocms.xyz/) (Also deployed with package changes)

  - [StudioCMS Ui Demo (ui-playground)](https://pr377.ui-testing.studiocms.xyz/)

File: packages/studiocms_core/src/schemas/config/index.ts [hunk 1]
```diff
@@ -74,6 +74,4 @@ export const StudioCMSOptionsSchema = z
 	.optional()
 	.default({});
 
-export type StudioCMSOptions = typeof StudioCMSOptionsSchema._input;
```
Comment (Adammatthiesen) [inline packages/studiocms_core/src/schemas/config/index.ts:77]: Why did you remove this? it is used by other components of the CMS

**Review by Adammatthiesen — CHANGES_REQUESTED** [review]

File: packages/studiocms_core/src/schemas/config/index.ts (hunk 1 above)
Comment (louisescher) [reply packages/studiocms_core/src/schemas/config/index.ts:77]: Knew I was gonna mess up during the rebase... lemme fix that really quick

**Review by Adammatthiesen — APPROVED** [review]

**Review by dreyfus92 — APPROVED** [review]
Thank you @louisescher, LGTM 🫡
